migration/mule/exposing-a-restful-resource-using-the-http-connector/README.md

```diff
@@ -0,0 +1,87 @@
+
+Follow the procedure below to create, then run the HTTP with JSON application.
+
+1. Download the GitHub Project and import the project to Integration Studio (File-> Import [WSO2-> Existing WSO2 projects into workspace])
+2. In the Package Explorer in Studio, right-click the project name, then select Run As > Run on Micro Integrator. The studio runs the application and WSO2 EI is up and kicking!
+3. Send a POST request to **http://localhost:8290/person** with the body equal to:
+		
+		{
+		 	"firstname":"John",
+		 	"lastname":"Doe",
+		 	"age": "12",
+		 	"address": 
+		  {
+		  	"streetAddress":"Lincoln St.",
+		    "city":"San Francisco",
+		    "state":"CA",
+		    "zipCode":"90401"
+			}
+		} 
+  You should receive a response saying a person was created successfully. For example: 
+
+		{ "personId": "1", "message": "Person was created"}
+
+4. Send a GET request to **http://localhost:8290/person**.
+  You should receive a response with all created persons.
+5. Take the value of `personId` from the previous response and send a GET request to **http://localhost:8290/person/{personId}**
+  You should recieve a response with the person data:
+
+		{"personId": "1", "firstname":"John","lastname":"Doe","address":{"streetAddress":"Lincoln St.","city":"San Francisco","state":"CA","zipCode":"90401"},"age":12}
+
+### How it Works
+
+The **Exposing a RESTful resource using the HTTP connector** example application consists of three, simple API Resources(https://ei.docs.wso2.com/en/latest/micro-integrator/develop/creating-artifacts/creating-an-api/) which receive end-user HTTP requests, set payloads based on the request paths, and send data and return the payloads to end-users as HTTP responses.
+
+The sections below elaborate further on the configurations of the application and how it works to respond to end-user requests.
+
+### POST Request Handling (1st Resource)
+
+When an end user's POST request reaches the application, first it will come to REST API's first resource(Which handles POST requests) [https://ei.docs.wso2.com/en/latest/micro-integrator/develop/creating-artifacts/creating-an-api/), which listens to POST requests at http://localhost:8290/person. 
+
+Next, the PayloadFactory create the following message to send back to the client: 
+
+	{ "personId": "1", "message": "Person was created"}
+
+Finally, WSO2 EI sends the message back to the client using [respond mediator](https://ei.docs.wso2.com/en/latest/micro-integrator/references/mediators/respond-Mediator/).
+
+### GET /person/{personId} Resource (2nd Resource)
+
+This resource is responsible for retrieving a specific person's data, based on the provided person ID. It starts with a WSO2 API Resource, which listens at http://localhost:8290/person/{personId}.
+
+Next, the flow will create a mock person detail using the PayloadFactory mediator and send the response using Respond mediator.
```

Comment:
Should we add a note here stating, that we are retuning a static payload?

Reply:
Updated